code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/EventUtils.java

```diff
@@ -71,13 +71,8 @@ static boolean isRequestIdentityEvent(final Event event) {
 	 * @param event the event to verify
 	 * @return true if urlVariables key is present in the event data
```

Comment:
Can you update the method description and this return statement to say true is returned if the urlvariables key is present and has a value of true?